modules/homepage/src/main/frontend/src/themePage/Navbars/headerStyle.jsx

```diff
@@ -109,7 +110,8 @@ const headerStyle = theme => ({
     color: whiteColor
   },
   aboveBackground: {
-    zIndex: "1301"
+    zIndex: "1301",
+    left: -theme.spacing(3) + "!important"
```

Comment:
This is invalid since it doesn't have a unit. Didn't test, but this should work:
```suggestion
    left: -theme.spacing(3) + "px !important"
```